engine/stagemaster/src/combo.rs

```diff
@@ -293,7 +293,17 @@ impl ComboListener {
     pub fn update(&mut self, event: &Event) {
         let old_current = self.current;
         let something_changed = match *event {
```

Comment:
Can we match on `event.event` to make the two following match arms simpler?

Reply:
fixed, thanks!